rocAL/include/readers/image/image_reader.h

```diff
@@ -81,13 +82,14 @@ struct ShardingInfo {
 struct ReaderConfig {
     explicit ReaderConfig(StorageType type, std::string path = "", std::string json_path = "",
                           const std::map<std::string, std::string> feature_key_map = std::map<std::string, std::string>(),
-                          bool shuffle = false, bool loop = false) : _type(type), _path(path), _json_path(json_path), _feature_key_map(feature_key_map), _shuffle(shuffle), _loop(loop) {}
+                          bool shuffle = false, bool loop = false, std::string index_path = "") : _type(type), _path(path), _json_path(json_path), _feature_key_map(feature_key_map), _shuffle(shuffle), _loop(loop), _index_path(index_path) {}
```

Comment:
can't we use the existing member for this? e.g, json_path

Reply:
Both are for different usecases. Would you remcommend a common name instead of specific paths (json/ index)

Reply:
Yes. Another option is to pass a vector of paths instead.